### DISCUSSION TITLE
Build Errors

### DISCUSSION THREAD

**Discussion by moodesu** [satnaing/astro-paper#419] [Q&A]
I have three errors when running 'pnpm run build' they all relate to the component <> being invalid.
Am I missing a dependency or something?
`src/layouts/PostDetails.astro:137:15 - error ts(2604): Component '< >' is not a valid component.

If this is a Svelte or Vue component, it might have a syntax error that makes it impossible to parse.

137               <>`

**Comment by man0war64** [comment]
I have the same problem... This might be related to the glob warnings in the other question? BTW. when I try to build astro-paper out of the box, I get the output below.

The <> might be related to the following?

11:42:46 [WARN] [glob-loader] Skipped 19 files that matched **/*.md.

It's a warning, but it also skips all the files... 



> dimensional-debris@4.7.0 build
> astro check && astro build

11:42:46 [content] Syncing content
11:42:46 [WARN] [glob-loader] The glob() loader cannot be used for files in src/content.
11:42:46 [WARN] [glob-loader] Skipped 19 files that matched **/*.md.
11:42:46 [content] Synced content
11:42:46 [types] Generated 243ms
11:42:46 [check] Getting diagnostics for Astro files in /Users/kriki/Blog/test/astro-paper/dimensional-debris...
src/components/Header.astro:89:21 - error ts(2604): Component '< >' is not a valid component.

If this is a Svelte or Vue component, it might have a syntax error that makes it impossible to parse.

89                     <>
                       ~~

src/layouts/Layout.astro:117:38 - warning ts(6133): 'rel' is declared but its value is never read.

117       onload="this.onload=null; this.rel='stylesheet';"
                                         ~~~
src/layouts/Layout.astro:117:20 - warning ts(6133): 'onload' is declared but its value is never read.

117       onload="this.onload=null; this.rel='stylesheet';"
                       ~~~~~~
src/layouts/Layout.astro:106:7 - warning astro(4000): This script will be treated as if it has the `is:inline` directive because it contains an attribute. Therefore, features that require processing (e.g. using TypeScript or npm packages in the script) are unavailable.

See docs for more details: https://docs.astro.build/en/guides/client-side-scripts/#script-processing.

Add the `is:inline` directive explicitly to silence this hint.

106       type="application/ld+json"
          ~~~~

src/layouts/PostDetails.astro:171:15 - error ts(2604): Component '< >' is not a valid component.

If this is a Svelte or Vue component, it might have a syntax error that makes it impossible to parse.

171               <>
                  ~~
src/layouts/PostDetails.astro:137:15 - error ts(2604): Component '< >' is not a valid component.

If this is a Svelte or Vue component, it might have a syntax error that makes it impossible to parse.

137               <>
                  ~~

Result (50 files):
- 3 errors
- 0 warnings
- 3 hints
